Tests/unittests.ps1

```diff
@@ -0,0 +1,24 @@
+#Requires -Modules @{ModuleName = 'PSSCriptAnalyzer'; ModuleVersion = '1.20.0'}
```

Comment:
Can we add a README to this directory about how to use these new scripts and write new tests? And link to it from the root README?

Reply:
Great idea - I'll update the docs!